sharing/examples/index.md

```diff
@@ -0,0 +1,46 @@
+---
+tags:
+  - sharing
+author:
+  display_name: UGRC Office
+  email: ugrc@utah.gov
+date: 2020-02-03 17:00:00
+title: SGID Sharing Highlights
+categories: []
+---
+
+![GIS Data Samples]({% link images/DataRequest.png %}){: style="width:50%; margin-left:auto; margin-right:auto; display:block" loading="lazy" }
+
+## Parcels - Collaboration
+{: .text-left}
+
+Our state-wide [parcel boundary dataset]({% link data/cadastre/parcels/index.html %}#UtahParcels) is the most-visited data page on our website, and for good reason: property boundaries (even their rough, un-surveyed approximations in GIS) are a foundational layer for many maps. They facilitate both commercial activity and private property management.
+
+This statewide collection would not be possible without the cooperation of counties across the state who send us updates to their parcel data on a regular basis. We merge them into a single, consistent hosted dataset available to anyone for free.
+
+## Roads - Collaboration
+{: .text-left}
+
+We are fortunate to have the luxury of free, up-to-date, and comprehensive [road data]({% link data/transportation/roads-system/index.html %}#RoadCenterlines). Thanks to the contributions of cities, counties, and UDOT, cartographers and analysts don't need to download road data from five different places with five different schemas, nor do they need to pay a commercial company to access data their tax dollars have already paid for. It's all here in one hosted dataset.
+
+## Land Ownership - Sharing
+{: .text-left}
+
+Sometimes a project needs a 10,000-foot overview of land ownership, especially in Utah where large amounts of land are owned by private individuals, the State, and the Federal Government. Utah has tasked the State Institutional Trust Lands Administration (SITLA) with maintaining a state-wide [land ownership layer]({% link data/cadastre/land-ownership/index.html%}#LandOwnership) as part of their responsibilities, which they share on their [website](https://trustlands.utah.gov/resources/maps/). Because this is such an important and fundamental dataset, SITLA also shares the data with the Open SGID and the SGID Open Data site for easy access to all.
```

Comment:
```suggestion
Sometimes a project needs a 10,000-foot overview of land ownership, especially in Utah where large amounts of land are owned by private individuals, the state, and the federal government. Utah has tasked the State Institutional Trust Lands Administration (SITLA) with maintaining a state-wide [land ownership layer]({% link data/cadastre/land-ownership/index.html%}#LandOwnership) as part of their responsibilities, which they share on their [website](https://trustlands.utah.gov/resources/maps/). Because this is such an important and fundamental dataset, SITLA also shares the data with the Open SGID and the SGID Open Data site for easy access to all.
```